napari/plugins/_npe2.py

```diff
@@ -271,8 +276,17 @@ def get_sample_data(
     return None, avail
 
 
-def _on_plugin_enablement_change(enabled: Set[str], disabled: Set[str]):
-    """Callback when any npe2 plugins are enabled or disabled"""
+def index_npe1_adapters():
```

Comment:
What's going on here? It doesn't look like this function is used in this file, do we still need?

Reply:
this is used in `menus/plugins_menu/PluginsMenu`.  the general pattern we've used here is to say that napari should be using stuff from `plugins/_npe2.py`, which in turn uses the npe2 api (as opposed to scattering our direct usage of the npe2 api across napari).  this isolation lets us focus our test coverage of the npe2 api on `_npe2.py`